routes/personal/personal.spec.js

```diff
@@ -74,17 +74,17 @@ describe('Test /personal responses', () => {
       expect(response.statusCode).toBe(422)
     })
 
-    test('it returns a 302 when selecting NO', async () => {
+    test('it returns a 302 when selecting unsupported province', async () => {
       const response = await request(app)
         .post('/personal/residence')
-        .send({ redirect: '/offramp', residence: 'No' })
+        .send({ redirect: '/offramp', residence: 'Alberta' })
       expect(response.statusCode).toBe(302)
```

Comment:
let's also check that it successfully redirected to the expected place, by adding this line right below here:
`expect(response.headers.location).toEqual('/offramp')`